web/src/presence.ts

```diff
@@ -24,6 +28,23 @@ export const presence_info_from_event_schema = z.object({
 });
 export type PresenceInfoFromEvent = z.output<typeof presence_info_from_event_schema>;
 
+const user_last_seen_response_schema = z.object({
+    result: z.string(),
+    msg: z.string().optional(),
+    presence: z
+        .object({
+            aggregated: z.object({
+                status: z.enum(["active", "idle", "offline"]),
+                timestamp: z.number(),
+            }),
+            website: z.object({
+                status: z.enum(["active", "idle", "offline"]),
+                timestamp: z.number(),
+            }),
```

Comment:
The `website` part here is not used or desired, so we can drop it from the type.